bambi/backends/pymc.py

```diff
@@ -279,9 +311,72 @@ def _laplace(model):
 
 def has_hyperprior(kwargs):
     """Determines if a Prior has an hyperprior"""
-
     return (
         "sigma" in kwargs
         and "observed" not in kwargs
         and isinstance(kwargs["sigma"], pm.model.TransformedRV)
     )
+
+
+def add_lkj(terms, eta=1):
+    # Parameters
```

Comment:
I would make this a full comment in numpy style, that way it shows up under `add_lkj.__doc__`

Reply:
Makes sense!